src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -638,8 +638,11 @@ smart_objects::SmartObjectSPtr MessageHelper::CreateDeviceListSO(
   smart_objects::SmartObject& list_so = (*device_list_so)[strings::device_list];
 
   int32_t index = 0;
+  // According to requirements, SDL should send info about 100 devices at
```

Comment:
@LitvinenkoIra 
I think code is self-descriptive enough, there no need to have that comment. Unfortunately we can't specify ticket in commit info however it is possible to find its hash on the GitHub and thus appropriate PR with link to requirement or defect.